spacy_llm/tasks/lemma/registry.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional, Type
+
+from ...registry import registry
+from ...ty import ExamplesConfigType, FewshotExample, TaskResponseParser
+from .task import LemmaTask
+from .util import DEFAULT_LEMMA_TEMPLATE_V1, LemmaExample, parse_responses_v1
```

Comment:
I think `util` is not a good name for this, as it's not just boilerplate utility code, this is specific functionality. Maybe a `parser.py` and `examples.py`. And read the templates in `task.py` (not married to any of these names/decisions though)

Reply:
Done in https://github.com/explosion/spacy-llm/pull/242/commits/bd416bafc2bf4461791dc666c790502af9dc430e.